### PR TITLE
DOC: Readme improvements for new users

### PR DESCRIPTION
**Pull request by abhijeetkalyan**
This PR adds a few explanatory comments to the README that might be helpful for users who are new to the Python/IPython ecosystem. It contains steps to get set up with a virtualenv(optional), PyMC3 and Theano(also optional) and a brief 'Usage' section to help users get started using pyfolio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,11 +55,37 @@ the IPython kernel when you make changes.
...
+```
+
 If you are on OSX and using a non-framework build of python you may need to set your backend:
 ``` bash
 echo "backend: TkAgg" > ~/.matplotlib/matplotlibrc
 ```
 
+## Usage
+
+A good way to get started is to run the examples in an [IPython notebook](https://ipython.org/notebook.html).
```
Comment (llllllllll) [inline README.md]: this has been renamed to the jupyter notebook and the command is now `jupyter notebook`

File: README.md [hunk 2]
```diff
@@ -27,6 +27,15 @@ See also [slides of a recent talk about pyfolio.](http://nbviewer.ipython.org/fo
 
 ## Installation
 
+### (Optional) Virtual Environment
+
+You might want to use a [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/) to avoid dependency conflicts between `pyfolio` and other python projects you have. To get set up with a virtual env, run:
+```bash
+
+mkvirtualenv pyfolio
```
Comment (richafrank) [inline README.md]: Is this meant for new users of pyfolio, or new developers of pyfolio?  I wouldn't expect to make a new virtualenv to use pyfolio with my existing project.

Comment (abhijeetkalyan) [inline README.md]: The latter, I'll clarify it here

File: README.md [hunk 3]
```diff
@@ -46,11 +55,37 @@ the IPython kernel when you make changes.
...
  - [zipline](https://github.com/quantopian/zipline) (optional; requires master, *not* 0.7.0)
 
+Some of Pyfolio's functionality, such as the [Bayesian tearsheet](https://github.com/quantopian/pyfolio/blob/master/pyfolio/examples/bayesian.ipynb), requires PyMC3 and Theano. To get set up, you can run:
+
+```bash
+pip install theano
+```
+
+```bash
+pip install --process-dependency-links git+https://github.com/pymc-devs/pymc3
```
Comment (twiecki) [inline README.md]: Actually this changed: https://github.com/pymc-devs/pymc3/pull/1109/files. Just remove `--process-dependency-links`.

Comment (abhijeetkalyan) [inline README.md]: Since that PR was merged about a minute ago, we're really keeping up with the state of the art :) updating

**Pull request merged**